collab-document/src/blocks/entities.rs

```diff
@@ -26,6 +28,10 @@ pub struct Block {
 pub struct DocumentMeta {
   /// Meta has a children map.
   pub children_map: HashMap<String, Vec<String>>,
+  /// Meta has a text map.
+  /// - @key: [Block]'s `external_id`
+  /// - @value: text delta json string - "\[ { "insert": "Hello World!", "attributes": { "bold": true } } \]"
+  pub text_map: Option<HashMap<String, String>>,
```

Comment:
I suppose the text_map should not be optional because there must be one text line in the document.

Reply:
In this situation, we need to consider the existing data in Flutter, as this text map does not exist in the current data unless data cleansing is performed.